### PR TITLE
Removed kubernetes specific dashboards, now each should work for ever…

### PR DESCRIPTION
**Pull request by Wiewiogr**
…y environment. Renamed metric `cluster` label to `namespace`

### CODE REVIEW

**Review by michalgawin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,14 +4,18 @@ This repo provides Grafana dashboards for reporting on Volt Active Data metrics
...
-The dashboards are provided in Grafana JSON format and are divided into three categories based on the Volt version and operating environment you are using:
+The dashboards are provided in Grafana JSON format and are divided into categories:
 
-- **Volt** - Dashboards for the latest version of Volt Active Data in any environment, including bare metal and VMs. These dashboards require an additional label `cluster` added as part of the Prometheus scraping process. (See below.)
-- **Volt-K8s** - Dashboards for the latest version of Volt Active Data running in Kubernetes.
-- **Volt-K8s-V10.x** - Dashboards for the long-term support (LTS) version 10 of Volt Active Data, running in Kubernetes.
+- **Volt** - Dashboards for the latest version of Volt Active Data in any environment, including bare metal, Kubernetes and VMs.
+- **Volt-V9.x** - Dashboards for the long-term support (LTS) version 9 of Volt Active Data, running in Kubernetes.
+- **Volt-V10.x** - Dashboards for the long-term support (LTS) version 10 of Volt Active Data, running in Kubernetes.
+- **Volt-V12.3.x** - Dashboards for the long-term support (LTS) version 12 of Volt Active Data, running in Kubernetes.
```
Comment (ajgent) [inline README.md:12]: Should be "Volt-V12.x" to match other LTS releases.

File: README.md [hunk 2]
```diff
@@ -4,14 +4,18 @@ This repo provides Grafana dashboards for reporting on Volt Active Data metrics
...
+- **Volt-V13.x** - Dashboards for the long-term support (LTS) version 13 of Volt Active Data, running in Kubernetes.
```
Comment (ajgent) [inline README.md:13]: Dashboards for the latest major release (V13,x) of Volt Active Data, running in Kubernetes.

Comment (ajgent) [reply README.md:13]: FYI -- there is no LTS release for V13 yet.

File: README.md [hunk 3]
```diff
@@ -4,14 +4,18 @@ This repo provides Grafana dashboards for reporting on Volt Active Data metrics
...
 
 Within each subdirectory are multiple dashboards, each providing a different view of the database activity, performance, or status. Try the different dashboards to see which ones suit your monitoring needs.
 
+Dashboards will work without any additional configuration inside Kubernetes, if you want to use them for Volt running in other environment, you will need an additional label `namespace` in the Volt metrics. (See below.)
```
Comment (ajgent) [inline README.md:17]: "Volt running in other environment," should be either of the following:

"Volt running in another environment,"
or
"Volt running in other environments,"

File: README.md [hunk 4]
```diff
@@ -42,7 +46,7 @@ scrape_configs:
     static_configs:
     - targets: ["localhost:1234"]
       labels:
-        cluster: 'local-volt'
+        namespace: 'local-volt'
```
[the remark this answers is not in the document]
Comment (Wiewiogr) [reply README.md:49]: It can be any string

**Pull request merged**